### DISCUSSION TITLE
ExcelJS + angular 12

### DISCUSSION THREAD

**Discussion by dittuxdev** [exceljs/exceljs#1738] [General]
It's this library compatible with angular 12? i whas making some test conding and i'm getting error when i try to run the project.
![exceljs](https://user-images.githubusercontent.com/54783343/120748836-0538a100-c4c9-11eb-80ee-3e70fcca999f.png)

**Comment by skypesky** [comment]
@dittuxdev Maybe I can help you.

update `tsconfig.app.json`
```
 "compilerOptions": {
    // add types
    "types": [
      "node"
    ]
  },
```
